src/styles.css

```
@@ -1,7 +1,31 @@
 @import "tailwindcss";
 @plugin "tailwindcss-primeui";
+@import "quill/dist/quill.core.css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)